src/store/BarrierStore.ts

```diff
@@ -275,8 +271,8 @@ export default class BarrierStore {
     }
 
     _fireOnBarrierChange = (): void => {
-        const high = this._high_barrier.visible ? +this._high_barrier.price.toFixed(this.pip) : undefined;
-        const low = this._low_barrier.visible ? +this._low_barrier.price.toFixed(this.pip) : undefined;
+        const high = this._high_barrier.visible ? +this._high_barrier.price : undefined;
+        const low = this._low_barrier.visible ? +this._low_barrier.price : undefined;
```

Comment:
should we make this change for accumulators?
This is breaking higher/lower contracts when we drag the barrier.